### ISSUE TITLE
About/Ecosystem (Distributors, Pro Services, Integrations, +more))

### ISSUE THREAD

**Issue by aijamalnk** [aijamalnk/istio.io-design-and-dev#8]
[Netlify link](https://deploy-preview-9187--istio-polidea.netlify.app/latest/about/ecosystem/)
Distributors
Pro Services
Integrations
Integrations + more

**Comment by wbartosz** [comment]
[Netlify link](https://deploy-preview-9070--istio-polidea.netlify.app/latest/about/ecosystem/)

**Comment by craigbox** [comment]
I think the font face is wrong in the boxes and Learn More buttons

**Comment by wbartosz** [comment]
> I think the font face is wrong in the boxes and Learn More buttons

You're right, well spotted. I'll add that along with the change David mentioned in the code review about the unused variable.

The changes should now be visible on [Netlify](https://deploy-preview-9070--istio-polidea.netlify.app/latest/about/ecosystem/).

**Comment by craigbox** [comment]
LGTM

**Comment by wbartosz** [comment]
[New netlify link with Istio logos](https://deploy-preview-9187--istio-polidea.netlify.app/latest/about/ecosystem/)

**Comment by craigbox** [comment]
Logos LGTM, thank you!
